server/protocols/webadmin/webadmin-cassandra-data/src/main/java/org/apache/james/webadmin/service/CassandraMappingsSolveInconsistenciesTask.java

```diff
@@ -19,17 +19,36 @@
 
 package org.apache.james.webadmin.service;
 
+import java.util.Optional;
+
 import javax.inject.Inject;
 
 import org.apache.james.rrt.cassandra.CassandraMappingsSourcesDAO;
 import org.apache.james.rrt.cassandra.migration.MappingsSourcesMigration;
 import org.apache.james.task.Task;
+import org.apache.james.task.TaskExecutionDetails;
 
 import reactor.core.publisher.Mono;
 
 public class CassandraMappingsSolveInconsistenciesTask implements Task {
     public static final String TYPE = "cassandraMappingsSolveInconsistencies";
 
+    public static class AdditionalInformation implements TaskExecutionDetails.AdditionalInformation {
```

Comment:
why do you still need this class? It does not thing out of duplicate `MappingsSourcesMigration.AdditionalInformation`'s information.